hbase_sql/src/main/java/com/splicemachine/derby/stream/spark/SparkOperationContext.java

```diff
@@ -455,6 +456,7 @@ public boolean isFailed(){
     public void setPermissive(String statusDirectory, String importFileName, long badRecordThreshold){
         this.permissive=true;
         this.badRecordThreshold = badRecordThreshold;
+	if(importFileName == null)importFileName=BAD_FILENAME + System.currentTimeMillis();
```

Comment:
Be careful here, to make sure that this file doesn't already exist; it's possible for system clocks to go backwards in time, and under enough load that would result in duplicates. I know, I know, it's highly unlikely, but it would also be an extremely hard bug to track down if it DID happen.